### DISCUSSION TITLE
Is there a cdn that provide the full compiled tailwind css file?

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#11736] [General]
Is there a cdn that provide the full compiled tailwind css file?

I need to include the css file in my template during development.

Using the <script> provided by playcdn will overwrite other css files.

**Comment by mybearworld** [comment]
This is not even possible because there are infinite classes.

**Comment by sokijaja** [comment]
I personally tried using this method in project recently and it was just so bad for the page load speed. A fully compiled tailwind.css file from an older version was about 1.2MB

**Reply by ghost** [reply]
I would say it's for local development use, so it's gonna be fine?

**Comment by fondoger** [comment]
@pinkfrog9  I created one repo for this purpose: https://github.com/fondoger/tailwindcss-cdn
I use Just-In-Time TailwindCSS along with AlpineJS. It is super good and easy to maintain because I only need to modify a single html file.

**Reply by ghost** [reply]
That is great. Wonder why tailwind does not provide it at first hand.

**Reply by ldalves** [reply]
I am sorry if this a basic question, but how to get full compiled tailwind css file from that? :)

**Reply by Guy1414** [reply]
I'm with you, how do we get the full compiled Tailwind CSS from that?

**Reply by mruijzendaal** [reply]
> I'm with you, how do we get the full compiled Tailwind CSS from that?

https://github.com/fondoger/tailwindcss-utilities

**Comment by code-kasha** [comment]
why don't you use cdn.tailwindcss.You can also extend it with it a tailwind.config,js
